designs/0032-cmdstan-output-formats.md

```diff
@@ -0,0 +1,230 @@
+- Feature Name: cmdstan-output-formats
+- Start Date: 2022-01-11
+- RFC PR:
+- Stan Issue:
+
+# Summary
+[summary]: #summary
+
+Provide alternatives to the [Stan CSV file format](https://mc-stan.org/docs/cmdstan-guide/stan_csv.html)
+for outputs of the Stan platform services methods supported by CmdStan.
+
+# Motivation
+[motivation]: #motivation
+
+The [Stan CSV file format](https://mc-stan.org/docs/cmdstan-guide/stan_csv.html) is limited and limiting for several reasons.
+
+The CSV format itself is not fully standardized.
+The original intent of the format is to represent a table's worth of data in plain text.
+Each row of the file contains one row of table data, and all rows have the same number of fields.
+Over time, conventions have been adopted, notably use of row headers for column labels and comment rows.
```

Comment:
I'm pretty sure most of this came in from day 1.  The original intent was to have it be a matrix of draws in CSV format if you ignored comments.  I have never liked the reading of the comments by CmdStan.